packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -187,7 +187,7 @@ describeCompat("Runtime IdCompressor", "NoCompat", (getTestObjectProvider, apis)
 		);
 	});
 
-	it("can normalize session space IDs to op space", async () => {
+	it.skip("can normalize session space IDs to op space", async () => {
```

Comment:
FYI - all these tests assume that ID compressor is not used by anybody else other than UT.
I think such tests should not be end-to-end tests, and I believe we have already equivalent stand-alone coverage.
I'm not sure I want to patch them, as this is only possible if ID compressor is used by other code before or after UT, but not in the middle of UT.

@taylorsw04 - FYI.

Reply:
Chatted with Taylor and he offered to follow up here after my change lands in main.